content/blog/2024-01-30-flash-framework.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: Alerting Systems are Incompatible with Modern Public Health Data Monitoring 
+author: Ananya Joshi
+date: 2024-01-01
+tags:
+  - flash
+authors:
+  - ajoshi
+heroImage: blog_alerts_hero.png
+heroImageThumb: blog_alerts_thumb.png
+output:
+  blogdown::html_page:
+    toc: true
+---
+Insights from public health data can keep communities safe. However, identifying these insights in large volumes of modern public health data can be laborious <sup>[1](#link1)</sup>. As a result, over the past few decades, public health agencies have built monitoring systems, like [ESSENCE](https://www.cdc.gov/nssp/new-users.html) (CDC), [EIOS](https://www.who.int/initiatives/eios) (WHO), and [DHIS2](https://dhis2.org/) (WHO), where users can set custom statistical alerts and then investigate these alerts using data visualizations <sup>[2](#link2)</sup>. These alerting systems largely follow the following formula <sup>[3](#link3)</sup> as shown in Fig 1.:
```

Comment:
In the netlify preview, there is weird spacing after each of the links (including the footnote references) in this line/paragraph.  I am not sure whats causing it...  Are there tabs hidden in here instead of spaces?  Is it related to your usage of the `<sup>` tags?  Maybe try [this carat notation](https://www.math.mcgill.ca/yyang/regression/RMarkdown/example.html#:~:text=To%20indicate%20a%20superscript%2C%20use,need%20to%20delimit%20with%20braces.) instead?

Reply:
I think it should be taken care of now - seems like a sup issue!